slasher/node/node.go

```diff
@@ -150,11 +150,11 @@ func (s *SlasherNode) Close() {
 	defer s.lock.Unlock()
 
 	log.Info("Stopping hash slinging slasher")
-	s.cancel()
```

Comment:
The same bug-prone code as in the beacon node - cancelling the parent context before `StopAll` is called.